README.md

```diff
@@ -0,0 +1,44 @@
+# TAD Conversion Tool
+
+This repository contains transformations to the TAD System Card. Currently supported transformations:
+1. Algoritmeregister v1.0.0 --> TAD System Card v0.1a3.
+1. Toetsingskader algoritmes v1 --> TAD Assessment Card v0.1a3.
+
+## Prerequisites
+On MacOS install [yq](https://mikefarah.gitbook.io/yq) by executing the following commmand
+in the terminal:
+```
+brew install yq
+```
+
+## Usage
+The directory `transformations/` contains the transformation files. These transformation files
+can be loaded by `yq` to transform a given input format to TAD System Cards. One can add transformations
+for new input formats.
+
+The directory `input/` contains the input files. Currently we support algoritmeregister v1.0.0 and
+toetsingskader algoritmes v1.
+
+### Algoritmeregister
+The `csv` file `input/algoritmeregister_v1.csv` contains all entries from the algoritmeregister with
+schema version v1.0.0. It is also possible to provide a single entry from the algoritmeregister instead
+of all the entries. These seperate entries can be downloaded from an algorithm page in the algoritmeregister.
+Make sure to only upload entries with schema version v1.0.0 because older versions require a different mapping.
+
+To convert all entries in the algoritmeregister with schema version v1.0.0 to TAD System Cards v0.1a3:
+```
+mkdir -p out/ && yq -p=csv -o=yaml --csv-auto-parse=f --from-file transformations/algoritmeregister/v100_to_systemcard_v01a3.yq  input/algoritmeregister_v1.csv | yq '.[]' -s '"out/"+(.owners[0].organization|downcase|sub(" "; "_")) + "_" + (.name|downcase|sub(" "; "_")|sub("/";"")|sub(":";"")) '
```

Comment:
Could we maybe have a `script/run` or something that runs this?

Reply:
Yes, it allows regex substitutes. Yes, will add a `scripts/run` file later on.